### PR TITLE
Removed hardcoded emotes from Ahune and moved them to DB

### PR DESCRIPTION
**Pull request by ChrisCGalbraith**
## Changes Proposed:
-  Added missing DB entries for Ahune (25740) to creature_text
-  Removed hard-coded text from boss_ahune.cpp

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4146

## Tests Performed:

- Tested in-game
- OS: Win10

## How to Test the Changes:

1. Enter Ahune event
2. Activate encounter and check that emotes occur

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1651953413104023300.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID` = 25740 AND `GroupID` = 0
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`)
+VALUES (25740, 0, 0, "Ahune Retreats. His defenses diminish.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RETREAT");
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 25740 AND `GroupID` = 1
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`)
+VALUES (25740, 1, 0, "Ahune will soon resurface.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RESURFACE");
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1651953413104023300.sql:8]: newline

Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1651953413104023300.sql:8]: turn the insert statements into 1 query only and the same with the delete

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1651953413104023300.sql]:
```sql
DELETE FROM `creature_text` WHERE `CreatureID` = 25740 AND `GroupID` IN (0,1);
INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
(25740, 0, 0, "Ahune Retreats. His defenses diminish.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RETREAT"),
(25740, 1, 0, "Ahune will soon resurface.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RESURFACE"); 
```

Comment (ChrisCGalbraith) [reply data/sql/updates/pending_db_world/rev_1651953413104023300.sql]: Will do

File: data/sql/updates/pending_db_world/rev_1651953413104023300.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID` = 25740 AND `GroupID` IN (0, 1);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(25740, 0, 0, "Ahune Retreats. His defenses diminish.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RETREAT"),
+(25740, 1, 0, "Ahune will soon resurface.", 41, 0, 100, 0, 0, 0, 0, 0, "ahune EMOTE_RESURFACE");
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1651953413104023300.sql:5]: Find broadcast_text and add the values.

Comment (ChrisCGalbraith) [reply data/sql/updates/pending_db_world/rev_1651953413104023300.sql:5]: Done

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Awesome work dude

**Pull request merged**